Source/DOM classes/Unported or Partial DOM/SVGImageElement.m

```diff
@@ -105,7 +106,7 @@ - (CALayer *) newLayer
 	 */
 	AppleNativeImageRef image = [AppleNativeImage imageWithData:imageData];
 	
-    if( image == nil ) // NSData doesn't contain an imageformat Apple supports; might be an SVG instead
+    if( image == nil && imageURL ) // NSData doesn't contain an imageformat Apple supports; might be an SVG instead
```

Comment:
Instead of:

 "&& imageURL"

let's change:

"if( effectiveSource == nil )"
to
"if( effectiveSource == nil && imageURL != nil )"

...effectiveSource will remain nil, but there's still the possibility for somoene to fix the iOS8 bug in SVGKSourceURL later